### PR TITLE
erd and two classes

### PR DESCRIPTION
**Pull request by guthb**
class for  Animal and Species, and refactored zoolandia erd

### CODE REVIEW

File: ZoolandiaRazor/Models/Animal.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.Linq;
+using System.Web;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+
+namespace ZoolandiaRazor.Models
+{
+    public class Animal
+    {
+        public int AnimailId { get; set; }
```
Comment (JZorvan) [inline ZoolandiaRazor/Models/Animal.cs]: Typo "AnimailId"

File: ZoolandiaRazor/Models/Animal.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+        public String  Name { get; set; }
```
Comment (JZorvan) [inline ZoolandiaRazor/Models/Animal.cs]: Extra space

**Comment by guthb** [comment]
need a check mark or thumbs up from ya'll to merge

**Review by ZachParris — APPROVED** [review]
👍

**Pull request merged**